.idea/modules.xml

```diff
@@ -3,11 +3,8 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/Backpack/Backpack.iml" filepath="$PROJECT_DIR$/Backpack/Backpack.iml" />
-      <module fileurl="file://$PROJECT_DIR$/Backpack/Backpack.iml" filepath="$PROJECT_DIR$/Backpack/Backpack.iml" />
```

Comment:
this file should be excluded from source control, right?

Reply:
There is a default gitignore when creating and android app, I think we're using that one and it's blacklisting a bunch of files from the `.idea` folder but not all of it
https://github.com/Skyscanner/backpack-android/blob/master/.gitignore#L39

Reply:
Ah that would imply that some of them should be in source control ... 🤔

Reply:
THat would be me guess but I might be completely wrong

Reply:
Also this https://stackoverflow.com/questions/26336709/android-studio-should-the-entire-idea-directory-be-in-git-ignore

Reply:
Wanna try removing it in a separate PR?

Reply:
I'll check if our gitignore is different from a default android one and change it accordgly 👍